controller.js

```diff
@@ -0,0 +1,129 @@
+const testsList = document.getElementById("testsList");
+
+
+
+
+document.getElementById("createTestButton").querySelector("img").addEventListener("click", () => createTest() );
+document.getElementById("sendDataButton").addEventListener("click", function () {
+  sendData();
+});
+
+
+
+function sendData() {
+let xhr = new XMLHttpRequest();
+let url = "DataProccesor";
+xhr.open("POST", url, true);
+xhr.setRequestHeader("Content-Type", "application/json");
+xhr.send(collectDataToJson());
+}
+
+function adjustInputFieldSize() {
+  let label = document.getElementById("searchInputField");
+  if (label.value.length > 5) {
+     label.cols = label.cols + 1;
+  }
```

Comment:
How does the `label.cols` supposed to get smaller, when I delete some characters?